### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following:

### DISCUSSION THREAD

**Discussion by Dolphin-in-river** [community/community#70008] [GitHub Education]
### Hi everyone,

I can't pass the pre-check for the student developer package. I have a valid student email address. I used to have a student git pro. However, now when I try to confirm the data again and join, I get this message.

Your application cannot be reviewed until you fix the following:
Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document.
Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

I have already entered the first name and second name in settings earlier and got the same error. Other people who reported this issue did not receive any answers or their problem was not resolved. My ticket is still open.

This person (https://github.com/orgs/community/discussions/69755) also has the same problem, and there are people with this problem in the comments.

**Comment by AndrewZzz24** [comment]
> ### Hi everyone,
> 
> 
> 
> I can't pass the pre-check for the student developer package. I have a valid student email address. I used to have a student git pro. However, now when I try to confirm the data again and join, I get this message.
> 
> 
> 
> Your application cannot be reviewed until you fix the following:
> 
> Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document.
> 
> Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.
> 
> 
> 
> I have already entered the first name and second name in settings earlier and got the same error. Other people who reported this issue did not receive any answers or their problem was not resolved. My ticket is still open.

Seems like a wide spread issue as some of my friends are facing same problem. I've also seen some threads here on git on this trouble. Would be happy if this post can be raised up so everyone gets a solution.

**Comment by GrzegorzZajac000** [comment]
Same here. I've got polish letter in my last name, but it doesn't work with and without it

**Reply by f4mno123** [reply]
Did it work for you in the end?, I have the same problem right now and have no idea how to fix it

**Reply by GrzegorzZajac000** [reply]
No, it didn't

**Comment by YuuSanLearnCode** [comment]
https://github.com/orgs/community/discussions/70242

this fix for me
